sdk/schemaregistry/schema-registry-avro/vitest.config.ts

```diff
@@ -8,7 +8,6 @@ export default mergeConfig(
   viteConfig,
   defineConfig({
     test: {
-      fileParallelism: false,
```

Comment:
Shouldn't we keep this since these tests most are mutating the registry?

Reply:
Fixed